src/Setup/WixBA/InstallationViewModel.cs

```diff
@@ -415,7 +415,8 @@ private void DetectComplete(object sender, DetectCompleteEventArgs e)
 
         private void PlanPackageBegin(object sender, PlanPackageBeginEventArgs e)
         {
-            if (WixBA.Model.Engine.StringVariables.Contains("MbaNetfxPackageId") && e.PackageId.Equals(WixBA.Model.Engine.StringVariables["MbaNetfxPackageId"], StringComparison.Ordinal))
+            // If this mba is running, we don't need to do anything with the NetFx4x package.
```

Comment:
I'd change this comment to more: "If we're able to run our BA, we don't want to install the .NET Framework since the framework on the machine is already good enough."